README.textile

```diff
@@ -154,6 +154,7 @@ bc.. # Required
 set :hipchat_token, "<your token>"
 set :hipchat_room_name, "Your room" # If you pass an array such as ["room_a", "room_b"] you can send announcements to multiple rooms.
 # Optional
+set :hipchat_enabled, true # set to false to prevent any messages from being sent
```

Comment:
@rberrelleza I'm not sure I understand what's wrong, what is in the doc are the defaults:

    set :hipchat_enabled, true # set to false to prevent any messages from being sent
    set :hipchat_announce, false # notify users


    def enabled?
      fetch(:hipchat_enabled, true)
    end

    def message_notification
      fetch(:hipchat_announce, false)
    end